python/pyiceberg/io/pyarrow.py

```diff
@@ -391,7 +391,7 @@ def visit_timestamp(self, _: TimestampType) -> pa.DataType:
         return pa.timestamp(unit="us")
 
     def visit_timestampz(self, _: TimestamptzType) -> pa.DataType:
-        return pa.timestamp(unit="us", tz="+00:00")
+        return pa.timestamp(unit="us", tz="UTC")
```

Comment:
Why change all uses to UTC rather than +00:00? Aren't these the same thing?

Reply:
Technically yes. https://github.com/apache/iceberg/issues/6945 has a deeper explanation for this. It is because the test data is written using a docker container that is using UTC.